video_xblock/static/js/videojs-tabindex.js

```diff
@@ -0,0 +1,42 @@
+domReady(function() {
+  videojs('{{ video_player_id }}').ready(function() {
+
+    // order tabIndex in player control
+    var orderTabIndex = function orderTabIndex(_player) {
+        var controlBarActions = Object.keys(_player.controlBar.childNameIndex_);
+        var controlsTabOrder = [
+            'progressControl',
+            'playToggle',
+            'playbackRateMenuButton',
+            'volumeMenuButton',
+            'resolutionSwitcher',
+            'fullscreenToggle',
+            'captionsButton'
+          ];
+        var controlsMap = {
+            'progressControl': _player.controlBar.childNameIndex_.progressControl.seekBar.el_,
```

Comment:
we can use a list of list to avoid 2 separate data structures
```js
[
   ['progressControl', _player.controlBar.childNameIndex_.progressControl.seekBar.el_], 
   [...]
]

```